frontend/src/Editor/Viewer.jsx

```diff
@@ -285,12 +285,13 @@ class ViewerComponent extends React.Component {
       .fetchAppBySlug(slug)
       .then((data) => {
         const isAppPublic = data?.is_public;
+        const released = data.current_version_id ? true : false;
```

Comment:
When we do this check. Lets say a version is release but the user needs to see the preview not the released app. so the both window title will be same. we have to show `Preview - <app-name> | Tooljet` . right ?

Can you please check those behaviours ?

Reply:
the current behaviour is correct.